examples/storm-starter/README.markdown

```diff
@@ -72,23 +72,6 @@ the Maven command to build and run storm-starter (see below), Maven will then be
 of Storm in this local Maven repository at `$HOME/.m2/repository`.
 
 
-## Running topologies with Maven
```

Comment:
@ptgoetz, what do you think of this change to remove these instructions?

What is the use case for running topologies using maven?